cmake/FindLibIntl.cmake

```diff
@@ -41,6 +41,16 @@ endif()
 if (MSVC)
   list(APPEND CMAKE_REQUIRED_LIBRARIES ${ICONV_LIBRARY})
 endif()
+
+# On macOS, if libintl is a static library then we also need
+# to link libiconv and CoreFoundation.
+get_filename_component(LibIntl_EXT "${LibIntl_LIBRARY}" LAST_EXT)
+if (APPLE AND (LibIntl_EXT STREQUAL ".a"))
```

Comment:
There really ought to be a better way to check if a library is static, but I'm too tired of CMake to work it out.

Reply:
I considered that, but I don't think the way we find `libintl` generates any targets...